### PR TITLE
subgraph doc fixes

### PR DESCRIPTION
**Pull request by conor2e**
## Description
Fixed the folllowing on the Subgraphs document:
- URLs for Deployment command and Querys updated to match latest version on prod.
- Details box at top of page changed to Warning.
- Email address changed to _support@chainstack.com_.
- SGR in deployment response URL changed to SG.

### CODE REVIEW

File: docs/subgraphs.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ meta:
 
 # Subgraphs
 
-::: details
```
Comment (akegaviar) [inline docs/subgraphs.md:11]: I suggest changing `::: warning` to `::: tip Information`

**Review by akegaviar — CHANGES_REQUESTED** [review]
Changes please

File: docs/subgraphs.md (hunk 1 above)
Comment (conor2e) [reply docs/subgraphs.md:11]: Hi @akegaviar. I'll assume I should do this for the other Tip lower in the page in the "Deploy a subgraph" section to be consistent. I'll also add it to our style guide as the standard for Tips that aren't "See also".

Comment (akegaviar) [reply docs/subgraphs.md:11]: Okay

**Review by akegaviar — APPROVED** [review]

**Review by ginstagram — APPROVED** [review]
LGTM

**Pull request merged**